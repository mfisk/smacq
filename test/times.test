print ts from pcapfile(0325@1112-snort.pcap) where ts > "30 years ago" and ts < "1 fortnight ago"
