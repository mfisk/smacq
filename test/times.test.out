2000-03-25T12:13:29.360668-07:00
2000-03-25T12:13:29.361618-07:00
2000-03-25T12:13:29.362882-07:00
2000-03-25T12:13:29.363329-07:00
2000-03-25T12:15:28.020237-07:00
2000-03-25T12:15:28.020780-07:00
2000-03-25T12:15:28.021223-07:00
2000-03-25T12:15:28.021391-07:00
2000-03-25T12:15:28.022503-07:00
2000-03-25T12:15:28.028445-07:00
2000-03-25T12:15:28.102582-07:00
2000-03-25T12:15:28.103886-07:00
2000-03-25T12:15:28.120922-07:00
2000-03-25T12:15:28.126184-07:00
2000-03-25T12:15:28.129182-07:00
2000-03-25T12:15:28.131837-07:00
2000-03-25T12:15:28.134177-07:00
2000-03-25T12:15:28.518317-07:00
2000-03-25T12:15:28.518393-07:00
2000-03-25T12:15:28.620389-07:00
2000-03-25T12:15:28.620464-07:00
2000-03-25T12:15:28.620554-07:00
2000-03-25T12:15:28.620627-07:00
2000-03-25T12:15:28.620701-07:00
2000-03-25T12:15:29.019047-07:00
2000-03-25T12:15:29.019123-07:00
2000-03-25T12:15:29.119180-07:00
2000-03-25T12:15:29.119256-07:00
2000-03-25T12:15:29.119330-07:00
2000-03-25T12:15:29.119403-07:00
2000-03-25T12:15:29.119476-07:00
2000-03-25T12:15:31.390280-07:00
2000-03-25T12:15:31.393042-07:00
2000-03-25T12:15:31.393132-07:00
2000-03-25T12:15:31.393278-07:00
2000-03-25T12:15:31.422224-07:00
2000-03-25T12:15:31.422384-07:00
2000-03-25T12:15:31.424355-07:00
2000-03-25T12:15:31.430712-07:00
2000-03-25T12:15:31.434037-07:00
2000-03-25T12:15:31.435816-07:00
2000-03-25T12:15:31.438199-07:00
2000-03-25T12:15:31.439352-07:00
2000-03-25T12:15:31.440841-07:00
2000-03-25T12:15:31.449118-07:00
2000-03-25T12:15:31.465803-07:00
2000-03-25T12:15:31.473742-07:00
2000-03-25T12:15:31.476108-07:00
2000-03-25T12:15:31.477595-07:00
2000-03-25T12:15:31.479025-07:00
2000-03-25T12:15:31.480107-07:00
2000-03-25T12:15:31.481168-07:00
2000-03-25T12:15:31.483377-07:00
2000-03-25T12:15:31.484744-07:00
2000-03-25T12:15:31.485619-07:00
2000-03-25T12:15:31.486659-07:00
2000-03-25T12:15:31.487872-07:00
2000-03-25T12:15:31.488971-07:00
2000-03-25T12:15:31.490265-07:00
2000-03-25T12:15:31.491501-07:00
2000-03-25T12:15:31.492601-07:00
2000-03-25T12:15:31.523056-07:00
2000-03-25T12:15:31.524317-07:00
2000-03-25T12:15:31.526237-07:00
2000-03-25T12:15:31.527459-07:00
2000-03-25T12:15:31.528634-07:00
2000-03-25T12:15:31.529772-07:00
2000-03-25T12:15:31.530822-07:00
2000-03-25T12:15:31.532031-07:00
2000-03-25T12:15:31.533792-07:00
2000-03-25T12:15:31.535031-07:00
2000-03-25T12:15:31.536216-07:00
2000-03-25T12:15:31.537292-07:00
2000-03-25T12:15:31.538362-07:00
2000-03-25T12:15:31.539540-07:00
2000-03-25T12:15:31.541175-07:00
2000-03-25T12:15:31.542351-07:00
2000-03-25T12:15:31.543539-07:00
2000-03-25T12:15:31.544603-07:00
2000-03-25T12:15:31.545646-07:00
2000-03-25T12:15:31.546804-07:00
2000-03-25T12:15:31.548553-07:00
2000-03-25T12:15:31.549749-07:00
2000-03-25T12:15:31.551027-07:00
2000-03-25T12:15:31.552164-07:00
2000-03-25T12:15:31.553174-07:00
2000-03-25T12:15:31.554210-07:00
2000-03-25T12:15:31.555230-07:00
2000-03-25T12:15:31.768206-07:00
2000-03-25T12:15:31.768626-07:00
2000-03-25T12:15:31.774992-07:00
2000-03-25T12:15:31.830889-07:00
2000-03-25T12:15:31.890978-07:00
2000-03-25T12:15:31.902515-07:00
2000-03-25T12:15:31.914011-07:00
2000-03-25T12:15:31.914651-07:00
2000-03-25T12:15:31.915711-07:00
2000-03-25T12:15:31.921672-07:00
2000-03-25T12:15:31.922436-07:00
2000-03-25T12:15:31.945492-07:00
2000-03-25T12:15:34.970249-07:00
2000-03-25T12:15:38.004917-07:00
2000-03-25T12:15:41.048696-07:00
2000-03-25T12:15:44.093205-07:00
2000-03-25T12:15:47.127418-07:00
2000-03-25T12:15:47.163434-07:00
2000-03-25T12:15:50.182853-07:00
2000-03-25T12:15:53.354234-07:00
